Shared/Supporting Files/Views/SampleList.swift

```diff
@@ -76,17 +86,29 @@ private extension SampleList {
                     VStack(alignment: .leading, spacing: 5) {
                         Text(sample.name)
                         if isShowingDescription {
-                            Text(sample.description)
-                                .font(.caption)
-                                .foregroundColor(.secondary)
-                                .transition(.move(edge: .top).combined(with: .opacity))
+                            if #available(iOS 16, *), deviceType != .phone {
+                                Text(sample.description)
+                                    .foregroundColor(selection == sample.name ? .white.opacity(0.8) : .secondary)
+                                    .font(.caption)
+                                    .transition(.move(edge: .top).combined(with: .opacity))
+                            } else {
+                                Text(sample.description)
+                                    .font(.caption)
+                                    .foregroundColor(.secondary)
+                                    .transition(.move(edge: .top).combined(with: .opacity))
+                            }
                         }
                     }
                     Spacer()
                     Button {
                         isShowingDescription.toggle()
                     } label: {
-                        Image(systemName: isShowingDescription ? "info.circle.fill" : "info.circle")
+                        if #available(iOS 16, *), deviceType != .phone {
+                            Image(systemName: isShowingDescription ? "info.circle.fill" : "info.circle")
+                                .foregroundColor(selection == sample.name ? .white : .accentColor)
+                        } else {
+                            Image(systemName: isShowingDescription ? "info.circle.fill" : "info.circle")
```

Comment:
Since both of these images are variants of one another, it would be better to specify the image name in one place and then use the `symbolVariant(_:)` modifier:

```swift
Image(systemName: "info.circle")
    .symbolVariant(isShowingDescription ? .fill : .none)
```